client/public/static/locales/pt/auth.json

```diff
@@ -1,44 +1,44 @@
+  "DISPLAY_NAME": "Nome de exibição",
+  "EMAIL": "Email",
+  "FORGOT_PASSWORD": "Esqueceu sua senha?",
+  "LOGIN": "Login",
+  "LOGOUT": "Logout",
+  "NEW_PASSWORD": "Nova senha",
+  "NO_ACCOUNT": "Não tem uma conta? Registe-se aqui.",
+  "NO_ACCOUNT_SIGNUP": "Não tem uma conta? <1>Registe-se aqui.</1>",
+  "OLD_PASSWORD": "Senha antiga",
+  "PASSWORD": "Senha",
+  "PASSWORD_CHANGE_SUCCESS": "Sua senha foi alterada com sucesso.",
+  "REMEMBER_ME": "Lembrar de mim na minha próxima visita",
+  "REQUEST_PASSWORD_RESET": "Solicitar redefinição de senha",
   "REQUEST_PASSWORD_RESET_EMAIL_SENT": {
-    "DESCRIPTION": "__STRING_NOT_TRANSLATED__",
-    "TITLE": "__STRING_NOT_TRANSLATED__"
+    "DESCRIPTION": "Enviamos um email. Siga o link para redefinir sua senha.",
+    "TITLE": "Sucesso"
   },
-  "RESEND_VERIFICATION_EMAIL": "__STRING_NOT_TRANSLATED__",
-  "RESET_PASSWORD": "__STRING_NOT_TRANSLATED__",
+  "RESEND_VERIFICATION_EMAIL": "Reenviar email de verificação",
+  "RESET_PASSWORD": "Redefina sua senha",
   "RESET_PASSWORD_SUCCESS": {
-    "DESCRIPTION": "__STRING_NOT_TRANSLATED__",
-    "TITLE": "__STRING_NOT_TRANSLATED__"
+    "DESCRIPTION": "Sua senha foi alterada com sucesso. Redirecionando...",
+    "TITLE": "Sucesso"
   },
-  "SIGN_UP": "__STRING_NOT_TRANSLATED__",
-  "THANKS_FOR_SIGNING_UP": "__STRING_NOT_TRANSLATED__",
+  "SIGN_UP": "Registe-se",
+  "THANKS_FOR_SIGNING_UP": "Obrigado por se registar!",
   "VALIDATION": {
-    "CONSECUTIVE_SEPARATOR_RULE": "__STRING_NOT_TRANSLATED__",
-    "DISPLAY_NAME_REQUIRED": "__STRING_NOT_TRANSLATED__",
-    "DISPLAY_NAME_RULES": "__STRING_NOT_TRANSLATED__",
-    "EMAIL_REQUIRED": "__STRING_NOT_TRANSLATED__",
-    "PASSWORDS_DO_NOT_MATCH": "__STRING_NOT_TRANSLATED__",
-    "PASSWORD_REQUIRED": "__STRING_NOT_TRANSLATED__",
-    "PASSWORD_RULES": "__STRING_NOT_TRANSLATED__",
-    "REQUIRED_FIELD": "__STRING_NOT_TRANSLATED__",
-    "START_END_RULE": "__STRING_NOT_TRANSLATED__",
-    "VALID_EMAIL": "__STRING_NOT_TRANSLATED__"
+    "CONSECUTIVE_SEPARATOR_RULE": "Não pode conter pontos finais consecutivos ou sublinhados",
+    "DISPLAY_NAME_REQUIRED": "Nome de exibição obrigatório",
+    "DISPLAY_NAME_RULES": "De 3 a 20 caracteres, consiste apenas de letras, números, pontos e sublinhados",
+    "EMAIL_REQUIRED": "Email obrigatório",
+    "PASSWORDS_DO_NOT_MATCH": "As duas senhas que você digitou não correspondem.",
+    "PASSWORD_REQUIRED": "Senha obrigatória",
+    "PASSWORD_RULES": "Mínimo de 8 caracteres, pelo menos uma letra minúscula, uma letra maiúscula e um número",
+    "REQUIRED_FIELD": "Campo obrigatório",
+    "START_END_RULE": "Não pode começar ou terminar com um ponto final ou sublinhado",
+    "VALID_EMAIL": "por favor digite um email válido."
   },
-  "VERIFICATION_MESSAGE": "__STRING_NOT_TRANSLATED__",
-  "WELCOME": "__STRING_NOT_TRANSLATED__"
+  "VERIFICATION_MESSAGE": "Enviamos um link de verificação para {{email}}. Por favor, verifique sua conta.",
+  "WELCOME": "Bem-vindo, "
```

Comment:
I think it's better using a more neutral way to display gender. I have translated it as "Bem-vindo (a)" in the past.